.ci/scripts/test_llama.sh

```diff
@@ -142,6 +151,9 @@ fi
 if [[ "${QE}" == "ON" ]]; then
   EXPORT_ARGS="${EXPORT_ARGS} --embedding-quantize 8,1024"
 fi
+if [[ "${MPS}" == "ON" ]]; then
+  EXPORT_ARGS="${EXPORT_ARGS} -kv -v --mps --disable_dynamic_shape"
```

Comment:
I am thinking of flipping this to explicitly op-in for dynamic_shape due to incrase in memory footprint

Reply:
😅